packages/@sanity/form-builder/src/inputs/common/fileTarget/utils/extractFiles.ts

```diff
@@ -0,0 +1,114 @@
+/**
+ * Utilities for extracting files from dataTransfer in a predictable cross-browser fashion.
+ * Also recursively extracts files from a directory
+ * Inspired by https://github.com/component/normalized-upload
+ * Contains special handling of
```

Comment:
of what

Reply:
good question, I started this sentence three years ago and i have no idea to be honest 😅